examples/nlp/text_classification_with_transformer.py

```diff
@@ -0,0 +1,184 @@
+"""
+Title: Text classification with Transformer
+Author: [Apoorv Nandan](https://twitter.com/NandanApoorv)
+Date created: 2020/05/10
+Last modified: 2020/05/10
+Description: Implement transformer block as a Keras layer and use it for text classification.
+"""
+"""
+## Setup
+"""
+
+import tensorflow as tf
+from tensorflow import keras
```

Comment:
Add `from tensorflow.keras import layers`, and use `layers.` whenever you use a layer object in your code